packages/ui/src/services/breakpoints/plugin/create-service.ts

```diff
@@ -0,0 +1,138 @@
+import { App, ref, Ref, computed, watch } from 'vue'
+
+import { warn } from '../../utils'
+import { getGlobalProperty } from '../../../vuestic-plugin/utils'
+import { addOrUpdateStyleElement } from '../../dom-functions'
+
+import { GlobalConfig } from '../../global-config/types'
+import { ThresholdsKeys, BreakpointsConfig, BodyClass } from '../types'
+
+export const createBreakpointsConfigPlugin = (app: App) => {
+  const globalConfig: Ref<GlobalConfig> | undefined = getGlobalProperty(app, '$vaConfig')?.globalConfig
+  if (!globalConfig) {
+    warn('createBreakpointsConfigPlugin: globalConfig is not defined!')
+    return {}
+  }
+
+  const breakpointsConfig: BreakpointsConfig | undefined = globalConfig.value.breakpoints
+  if (!breakpointsConfig) {
+    warn('createBreakpointsConfigPlugin: breakpointsConfig is not defined!')
+    return {}
+  }
+
+  if (!breakpointsConfig.enabled) {
+    return {}
+  }
+
+  if (!breakpointsConfig.thresholds || !Object.values(breakpointsConfig.thresholds).length) {
+    warn('createBreakpointsConfigPlugin: there are no defined thresholds!')
+    return {}
+  }
+
+  const isMounted = computed(() => !!document.documentElement)
```

Comment:
useMounted

Reply:
It uses `onMounted` hook and it can't be used outside `setup` function.

Reply:
document will be undefined in SSR. Use SSR utils here.